### PR TITLE
add cpu and memory request for jwtproxy

### PR DESCRIPTION
**Pull request by sparkoo**
Signed-off-by: Michal Vala <mvala@redhat.com>

### What does this PR do?
Adds new config properties for jwtproxy resources requests:
```
che.server.secure_exposer.jwtproxy.memory_request=15mb
che.server.secure_exposer.jwtproxy.cpu_request=0.03
```

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/18483

### How to test this PR?

1. start any workspace
1. take a look at jwtproxy pod mem and cpu requests

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by skabashnyuk — APPROVED** [review]

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties [hunk 1]
```diff
@@ -606,7 +606,9 @@ che.server.secure_exposer.jwtproxy.token.issuer=wsmaster
 che.server.secure_exposer.jwtproxy.token.ttl=8800h
 che.server.secure_exposer.jwtproxy.auth.loader.path=/_app/loader.html
 che.server.secure_exposer.jwtproxy.image=quay.io/eclipse/che-jwtproxy:0.10.0
+che.server.secure_exposer.jwtproxy.memory_request=10mb
```
Comment (ibuziuk) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties]: LGTM, but taking a look at the metrics at OSD v4 maybe increasing request to 15mb to be on the safe side

image

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties [hunk 2]
```diff
@@ -606,7 +606,9 @@ che.server.secure_exposer.jwtproxy.token.issuer=wsmaster
...
 che.server.secure_exposer.jwtproxy.memory_limit=128mb
+che.server.secure_exposer.jwtproxy.cpu_request=0.02
```
Comment (ibuziuk) [inline assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties]: Also, might be good to increase the request to `0.03`

image

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties (hunk 1 above)
Comment (sparkoo) [reply assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties]: updated

File: assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties (hunk 2 above)
Comment (sparkoo) [reply assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties]: updated

**Review by mshaposhnik — APPROVED** [review]

**Review by ibuziuk — APPROVED** [review]
:+1:

**Pull request merged**